libminifi/src/utils/StringUtils.cpp

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
The content of this file is mostly moved from the header, the changes I made were to make linter happy. Some of these functions still leave space for improvement.

Reply:
hmm, can we minimize those changes? Like the idea but it seems that there is scope creep. Is there an actual motivation behind another ticket for which this is necessary? I think I'm -1 on general cleanup until the follow on release since we have so much to test in this release.

Reply:
quite nice.